conda/models/environment.py

```diff
@@ -0,0 +1,151 @@
+# Copyright (C) 2012 Anaconda, Inc
+# SPDX-License-Identifier: BSD-3-Clause
+"""EXPERIMENTAL Conda environment data model"""
+
+from __future__ import annotations
+
+from dataclasses import dataclass, field
+from logging import getLogger
+from typing import TYPE_CHECKING
+
+from ..exceptions import CondaValueError
+
+if TYPE_CHECKING:
+    from typing import Any
+
+    from .match_spec import MatchSpec
+    from .records import PackageRecord
+
+
+log = getLogger(__name__)
+
+
+@dataclass
+class Environment:
+    """
+    **Beta** While in beta, expect both major and minor changes across minor releases.
+
+    Data model for a conda environment.
+    """
+
+    #: Prefix the environment is installed into (required).
+    prefix: str
+
+    #: The platform this environment may be installed on (required)
+    platform: str
+
+    #: Environment level configuration, eg. channels, solver options, etc.
+    config: dict[str, Any] = field(default_factory=dict)
+
+    #: Map of other package types that conda can install. For example pypi packages.
+    external_packages: dict[str, list[Any]] = field(default_factory=dict)
```

Comment:
```suggestion
    external_packages: dict[str, list[str]] = field(default_factory=dict)
```

Are we sure this is Any?